src/radius/radius.c

```diff
@@ -5,19 +5,14 @@
  * This software may be distributed under the terms of the BSD license.
  * See README for more details.
```

Comment:
I'm guessing you re-synced this with some other project? Can you add details of which project and which commit/tag you got these changes from?

Reply:
ok.